src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -1294,36 +1294,59 @@ public LazyConditionalChoiceExpression(Lazy<IOperation> condition, Lazy<IOperati
     /// <summary>
     /// Represents a conversion operation.
     /// </summary>
-    internal abstract partial class BaseConversionExpression : Operation, IHasOperatorMethodExpression, IConversionExpression
+    internal abstract partial class BaseConversionExpression<TConversion> : Operation, IHasOperatorMethodExpression, IConversionExpression
+        where TConversion : struct, IConversion
```

Comment:
It provides the visitor implementation that both languages use, plus any common properties that don't need to be reimplemented. I'd rather leave it.